pkgs/servers/monitoring/prometheus/json-exporter.nix

```diff
@@ -0,0 +1,24 @@
+# This file was generated by go2nix.
+{ stdenv, buildGoPackage, fetchgit, lib }:
+
+buildGoPackage rec {
+  name = "prometheus-json-exporter-${version}";
+  version = "20160913-${stdenv.lib.strings.substring 0 7 rev}";
```

Comment:
Please use the following version scheme for packages with releases: `prometheus-json-exporter-unstable-2016-09-13`